lib/src/user.dart

```diff
@@ -92,4 +94,68 @@ class User {
     await _channel
         .invokeMethod('setUserTag', {"attributeKey": key, "mpid": this.mpid});
   }
+
+  /// Returns all GDPR Consent states with their purpose.
+  Future<Map> getGDPRConsentState() async {
```

Comment:
I think we should import GDPRConsent and set this to return:
```suggestion
  Future<GDPRConsent> getGDPRConsentState() async {
```